servers/zms/src/main/java/com/yahoo/athenz/zms/store/impl/jdbc/JDBCConnection.java

```diff
@@ -476,6 +477,7 @@ public class JDBCConnection implements ObjectStoreConnection {
     private static final String SQL_INSERT_PENDING_GROUP_MEMBER = "INSERT INTO pending_principal_group_member "
             + "(group_id, principal_id, expiration, audit_ref, req_principal) VALUES (?,?,?,?,?);";
     private static final String SQL_DELETE_GROUP_MEMBER = "DELETE FROM principal_group_member WHERE group_id=? AND principal_id=?;";
+    private static final String SQL_DELETE_EXPIRED_GROUP_MEMBER = "DELETE FROM principal_group_member WHERE group_id=? AND principal_id=? AND ? = expiration;";
```

Comment:
let's update this to have "expiration=?" to be consistent with other where conditions